.gitignore

```
@@ -59,6 +59,7 @@ nosetests.xml
 
 # Sphinx documentation
 docs/_build/
+projects/orquestra-sdk/docs/_build
```

We don't keep `docs` at the repo root nowadays
```suggestion
projects/orquestra-sdk/docs/_build
```